docs/book/src/index.md

```diff
@@ -30,7 +30,7 @@ Whilst Rust is a great systems programming language (and Sway itself is written
 
 Rust shines because it can use zero-cost abstractions and its sophisticated borrow-checker memory model to achieve impressive runtime performance for complex programs without a garbage collector.
 
-On a blockchain, cost of execution and deployment is the scarce resource.
+On a blockchain, cost of execution and deployment is scarce.
```

Comment:
This suggestion doesn't really make sense either. The cost isn't scarce, it's just costly.

```suggestion
On a blockchain, execution and deployment are limited by transaction costs.
```

Reply:
There may be a better term than cost to analogize with space and time, which are resources.